web/modules/custom/ai_screening_project/src/Template/ProjectStatusEvaluationTwigExtension.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Drupal\ai_screening_project\Template;
+
+use Drupal\ai_screening_project\Helper\ProjectHelper;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+/**
+ * Twig extension providing project related functionalities.
+ *
+ * @package Drupal\ai_screening_project\Template
+ */
+class ProjectStatusEvaluationTwigExtension extends AbstractExtension {
```

Comment:
Consider renaming to make the name more generic (we may add more functions and stuff in the future):

```suggestion
class ProjectTwigExtension extends AbstractExtension {
```